### DISCUSSION TITLE
Support for multiple files-bundling of yaml files

### DISCUSSION THREAD

**Discussion by Mhaavald** [microsoft/OpenAPI.NET#883] [Q&A]
Hi,

the OpenApiStreamReader does not deal with the case where the yaml fil has references to multiple files with definitions.
This is a common way of structuring the yaml file into more a maintainable structure, so I wonder whether anyone knows how to bundle yaml files into one, or support these references as part of  OpenApiStreamReader?

/Morten

**Comment by darrelmiller** [comment]
The current release will handle external references to files that are complete OpenAPIDescriptions.  We do not yet support external references to fragments of JSON/YAML.  There is a reader setting that allows inlining of external references.
